src/perl/lib/wtsi_clarity/dao/flgen_plate_dao.pm

```diff
@@ -127,8 +128,8 @@ sub _build_well {
 
   # study
   my $study = $self->_get_study($sample->project_limsid);
-  $well{'id_study_lims'} = $study->id;
-  $well{'cost_code'}     = $study->cost_code;
+  $well{'study_id'}  = $study->id;
```

Comment:
I can't see `study_id` either in the required JSON message or in the DB schema.
In the JSON there is a `study_uuid` (but that is a defined as a constant in the perl module) or in the DB schema there is a `id_study_tmp` field.